x-pack/plugins/snapshot_restore/public/app/sections/home/home.tsx

```diff
@@ -9,7 +9,7 @@ import { Route, RouteComponentProps, Switch } from 'react-router-dom';
 
 import { EuiPageBody, EuiPageContent, EuiSpacer, EuiTab, EuiTabs, EuiTitle } from '@elastic/eui';
 
-import { BASE_PATH, Section } from '../../constants';
+import { BASE_PATH, getHomeBreadcrumb, Section } from '../../constants';
```

Comment:
It looks strange to me to import a method from a constants file...

To me, like we quickly discuss over slack, we should save a reference to `i18n` in some sort of `TranslateService` file, so we don't need to pass it around from the app dependencies from inside a component.

Here, if we are going to update the breadcrumb many times, I'd have a `BreadcrumbsService` with a simple api

```ts
import { breadCrumbsService } from '../services/breadcrumbs'`

breadCrumbsService.set('home'); // all the logic and translation (getHomeBreadcrumb, getRepositoryAddBreadcrumb) is done there
```